### PR TITLE
Add window titles

### LINKED ISSUE #2775: Dashboard 2.0: Update window title when views change
Currently the window title is always the generic "Dashboard" so it's impossible to tell two tabs apart. We should have key components update the window title when they are mounted.

Could we use the PageHeader for this?

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it adds window titles to dashboard's app.
Resolves #2775 

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: saleor/static/dashboard-next/orders/views/OrderDetails.tsx [hunk 1]
```diff
@@ -337,264 +338,278 @@ export const OrderDetails: React.StatelessComponent<OrderDetailsProps> = ({
...
-                                  }))}
-                                  shippingMethods={maybe(
-                                    () => data.order.availableShippingMethods,
-                                    []
-                                  )}
-                                  onOrderCancel={variables =>
-                                    orderCancel.mutate({
+                          <>
+                            <WindowTitle
+                              title={maybe(() => "#" + data.order.number)}
```
Comment (maarcingebala) [inline saleor/static/dashboard-next/orders/views/OrderDetails.tsx]: I think "Order #{number}" would be better here.

File: saleor/static/dashboard-next/products/views/ProductVariant.tsx [hunk 2]
```diff
@@ -104,42 +105,47 @@ export const ProductVariant: React.StatelessComponent<ProductUpdateProps> = ({
...
+                              updateVariant.mutate({
+                                attributes: data.attributes
+                                  ? data.attributes
+                                  : null,
+                                costPrice: decimal(data.costPrice),
+                                id: variantId,
+                                priceOverride: decimal(data.priceOverride),
+                                quantity: data.quantity || null,
+                                sku: data.sku,
+                                trackInventory: true // FIXME: missing in UI
```
Comment (maarcingebala) [inline saleor/static/dashboard-next/products/views/ProductVariant.tsx]: Btw. is it still missing in UI? :)

File: saleor/static/dashboard-next/orders/views/OrderDetails.tsx (hunk 1 above)
Comment (maarcingebala) [reply saleor/static/dashboard-next/orders/views/OrderDetails.tsx]: Also, I think it might be useful to display "Draft order #{number}" for drafts.

Comment (dominik-zeglen) [reply saleor/static/dashboard-next/orders/views/OrderDetails.tsx]: Sure, on it

File: saleor/static/dashboard-next/products/views/ProductVariant.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/products/views/ProductVariant.tsx]: Yes... I think we can add checkbox in 'Inventory' card, like "Charge taxes for this item" in "Pricing" card.

zrzut ekranu 2018-11-13 o 17 09 05

**Pull request merged**